### PR TITLE
Create database and grant privileges

### PR DESCRIPTION
**Pull request by celestian**
RP Crucial Mode cannot run without database. This is the first step: creation of database for RPCM with proper charset, collation and privileges granted to RPCM DB user.

There other issue [Enable RPCM DB in docker](https://github.com/rpcm-org/docker-rpcm/issues/14) which does the necessary work on docker side.

Own database schema will be created in next PR.

Note: Right now, there is now quick way how to test together PR from rpcm and docker-rpcm project. So, both PRs doesn't break current functionality as long as there is no real functionality. Testing will be done after merge.

### CODE REVIEW

**Comment by celestian** [comment]
@redoper1 please, wait with merge, I found out how to test it. And I am afraid, I miss something. I will let you know when it is ready.

File: sql/grant_user_to_database.sql.in [hunk 1]
```diff
@@ -0,0 +1,3 @@
+CREATE USER '{RPCM_DB_USER}'@'{RPCM_LOCAL_IP}' IDENTIFIED BY '{RPCM_DB_PASS}';
+GRANT ALL ON {RPCM_DB_NAME}.* TO '{RPCM_DB_USER}'@'{RPCM_LOCAL_IP}';
```
Comment (redoper1) [inline sql/grant_user_to_database.sql.in]: I think we should grant the user only needed permissions, don't you think so @celestian?

File: sql/create_database.sql.in [hunk 2]
```diff
@@ -0,0 +1,3 @@
+CREATE DATABASE czech_slovak_names
```
Comment (redoper1) [inline sql/create_database.sql.in]: If I get it right, that should be name for whole database for RPCM? Maybe we can think about something more appropriative.

Comment (celestian) [reply sql/create_database.sql.in]: Wau! Nice catch! This was from example :-)

**Comment by celestian** [comment]
@redoper1 PR is now ready for review.

File: sql/grant_user_to_database.sql.in (hunk 1 above)
Comment (celestian) [reply sql/grant_user_to_database.sql.in]: Addressed.

**Pull request merged**